framework/include/utils/RankFourTensorImplementation.h

```diff
@@ -542,62 +541,151 @@ RankFourTensorTempl<T>::fillFromInputVector(const std::vector<T> & input, FillMe
 
 template <typename T>
 void
-RankFourTensorTempl<T>::fillSymmetricFromInputVector(const std::vector<T> & input, bool all)
+RankFourTensorTempl<T>::fillSymmetric9FromInputVector(const std::vector<T> & input)
 {
-  if ((all == true && input.size() != 21) || (all == false && input.size() != 9))
-    mooseError("Please check the number of entries in the stiffness input vector.");
+  if (input.size() != 9)
+    mooseError(
```

Comment:
It would be appropriate to make this a `mooseAssert` (i.e. check only in debug mode).

Reply:
I changed a few to asserts, should I have changed all of them?